gitpod-io/gitpod
# [ws-manager-mk2] Support public SSH keys

## Description
Support public ssh keys

## Related Issue(s)
n.a.

## How to test
- Create ssh keys with `ssh-keygen -t ed25519` 
- Add the ssh key in user settings
- Start workspace
- Check that the workspace CR sshPublicKeys property is set
- Check that the `gitpod.io/sshPublicKeys` annotation is set on the workspace pod
- Create second ssh key
- Add the ssh key in user settings
- Check that workspace CR and workspace pod have been updated

## Release Notes

```release-note
None
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
